$MirOS$

	cf. Message-ID: <Pine.BSM.4.64L.0705201013380.8619@odem.66h.42h.de>
	and the following thread

--- src/LYUtils.c.orig	Thu May 17 22:52:59 2007
+++ src/LYUtils.c	Sun May 20 11:17:22 2007
@@ -5367,8 +5367,7 @@ char *LYTildeExpand(char **pathname,
     if (LYIsTilde(temp[0])) {
 
 	CTRACE((tfp, "LYTildeExpand %s\n", *pathname));
-	if (LYIsPathSep(temp[1])
-	    && temp[2] != '\0') {
+	if (LYIsPathSep(temp[1])) {
 	    char *first = NULL;
 	    char *second = NULL;
 
